src/rimestate.cpp

```diff
@@ -440,6 +441,30 @@ void RimeState::updateUI(InputContext *ic, bool keyRelease) {
 
 void RimeState::release() { session_.reset(); }
 
+void RimeState::commitInput(InputContext *ic) {
+    if (auto *api = engine_->api()) {
+        if (auto input = api->get_input(this->session())) {
```

Comment:
please use visible type instead of auto here.

Reply:
Just this line, or all `RimeState::commit*` functions (including untouched `RimeState::commitPreedit`)?

Reply:
Just this line